### ISSUE TITLE
Exclude Gmail

### ISSUE THREAD

**Issue by maverick74** [containers-everywhere/contain-google#52]
Would be great to be able to exclude Gmail, the same way one can exclude Maps.

**Comment by krisu5** [comment]
This is only reason I can't use this addon.

**Comment by maverick74** [comment]
Maybe most consider themselves feature complete...

**Comment by Perflyst** [comment]
Feel free to add it, you can see [here](https://github.com/containers-everywhere/contain-google/commit/1fc060323f869fa58f4a33dcf8a1dbe06e48e063) how it can be done.

This addon is to **contain** google and not to login to Google outside the container so that Google can easily identify you based on third party cookies etc

**Comment by maverick74** [comment]
> Feel free to add it, you can see [here](https://github.com/containers-everywhere/contain-google/commit/1fc060323f869fa58f4a33dcf8a1dbe06e48e063) how it can be done.
> 
> This addon is to **contain** google and not to login to Google outside the container so that Google can easily identify you based on third party cookies etc

You make a great point and I totally understand and agree but...

Why are there other exceptions to other google services then?

**Comment by Tanman897** [comment]
Yeah, I saw the exclusion area in the options tab for this extension. I added "mail.google.com" since it would not accept the full address or "/" or anything but this. This creates a 400 error every time that I now try to access Gmail and it essentially breaks any mail.google domain access. I find it really weird that the exceptions option doesn't accept full urls and does not accept "/" or offer any way to exclude a wildcard domain of sorts (i.e. mail.google.com/whatever-after-this.
